morpcc/navigator.py

```diff
@@ -92,13 +92,13 @@ def add_attribute(
 
     def add_relationship(
         self,
-        name,
-        title,
+        name: str,
+        title: str,
         reference_attribute,
         reference_search_attribute,
```

Comment:
missing type annotation

Reply:
How do I know this type?
In create_app.py: 
reference_attribute=cifid.attributes["cifid"],
reference_search_attribute=cifid.attributes["cifid"]

Reply:
from studying the code, what type you think output of cifid.attributes['cifid']?

Reply:
you can also try including breakpoint() before that function call in create_app.py .. and try to run the script using morpfw execute .. that'll allow you to inspect the contents